compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/BIRPackageSymbolEnter.java

```diff
@@ -849,18 +889,18 @@ private PackageID createPackageID(String orgName, String pkgName, String pkgVers
     }
 
     private class BIRTypeReader {
-        static final int SERVICE_TYPE_TAG = 51;
+        public static final int SERVICE_TYPE_TAG = 51;
```

Comment:
If this constant is to be used outside of this class wouldn't it be better to move it to a relevant `Contants` class instead of making in public?